### ISSUE TITLE
DBeaver fails at launch: Unrecognized VM Options?

### ISSUE THREAD

**Issue by Pandorius** [dbeaver/dbeaver#3119]
I recently installed DBeaver 5.0.0. It worked fine for a couple of days but it now doesn't want to launch and it throws me the following message: 

![capture2](https://user-images.githubusercontent.com/24669946/37210943-af9464fe-23aa-11e8-8e25-a2dc708652b3.PNG)

I have no idea about how to solve this...

**Comment by ruspl-afed** [comment]
Probably you are trying to run 64-bit DBeaver with 32-bit JRE.
You can try answers from this thread https://stackoverflow.com/questions/27019786/eclipse-java-was-started-but-returned-error-code-13

**Comment by ruspl-afed** [comment]
Anything else in the .log?

**Comment by Pandorius** [comment]
I'm afraid not :/

**Comment by serge-rider** [comment]
@Pandorius could you try to install DBeaver using windows installer (64 or 32 bit)?

**Comment by Pandorius** [comment]
I uninstalled and reinstalled DBeaver 5.0.0 with the same .exe file and it now works... Computers are definitely strange beings :)

But, as said earlier, my DBeaver 5.0.0 had been working fine for a couple of days before it started popping up error messages. I hope it'll work for good this time.

Thank you!

**Comment by Pandorius** [comment]
I rebooted my computer and now it's back... :(

![capture](https://user-images.githubusercontent.com/24669946/37326183-421c5870-2691-11e8-8ff6-b8719a92a6b7.PNG)

> !SESSION 2018-03-13 07:33:58.927 -----------------------------------------------
> eclipse.buildId=unknown
> java.version=9.0.4
> java.vendor=Oracle Corporation
> BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
> Command-line arguments:  -os win32 -ws win32 -arch x86_64
> !ENTRY org.eclipse.osgi 4 0 2018-03-13 07:33:59.253
> !MESSAGE Application error
> !STACK 1
> java.lang.IllegalStateException: Unable to acquire application service. Ensure that the org.eclipse.core.runtime bundle is resolved and started (see config.ini).
> 	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:78)
> 	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
> 	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
> 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
> 	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:653)
> 	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:590)
> 	at org.eclipse.equinox.launcher.Main.run(Main.java:1499)
> 	at org.eclipse.equinox.launcher.Main.main(Main.java:1472)
>

**Comment by ruspl-afed** [comment]
It looks something fights with DBeaver on your PC. It is interesting what do you have inside  dbeaver/plugins directory after restart.

**Comment by Pandorius** [comment]
I'm not totally sure what you mean... Here is what I've got inside _C:\Program Files (x86)\DBeaver\plugins_:

22-02-18  10:17            52.352 org.apache.commons.cli_1.3.1.jar
22-02-18  10:17           350.116 org.apache.felix.scr_2.0.10.v20170501-2007.jar
22-02-18  10:17            59.165 org.eclipse.compare.core_3.6.100.v20170516-0820.jar
18-02-18  17:31             1.252 org.eclipse.core.commands.nls_3.9.0.v20170530-1048.jar
22-02-18  10:17           102.207 org.eclipse.core.commands_3.9.0.v20170530-1048.jar
22-02-18  10:17            93.184 org.eclipse.core.contenttype_3.6.0.v20170207-1037.jar
18-02-18  17:31             5.872 org.eclipse.core.expressions.nls_3.6.0.v20170207-1037.jar
22-02-18  10:17            80.793 org.eclipse.core.expressions_3.6.0.v20170207-1037.jar
18-02-18  17:31             9.324 org.eclipse.core.filebuffers.nls_3.6.100.v20170203-1130.jar
22-02-18  10:17           107.576 org.eclipse.core.filebuffers_3.6.100.v20170203-1130.jar
22-02-18  10:17            96.952 org.eclipse.core.jobs_3.9.2.v20171030-1027.jar
22-02-18  10:17           830.379 org.eclipse.core.resources_3.12.0.v20170417-1558.jar
18-02-18  17:31             9.419 org.eclipse.core.runtime.nls_3.13.0.v20170207-1030.jar
22-02-18  10:17            66.849 org.eclipse.core.runtime_3.13.0.v20170207-1030.jar
18-02-18  17:31           126.188 org.eclipse.debug.ui.nls_3.12.50.v20170920-1329.jar
22-02-18  10:17         2.910.263 org.eclipse.debug.ui_3.12.50.v20170920-1329.jar
22-02-18  10:17            18.305 org.eclipse.e4.core.commands_0.12.100.v20170513-0428.jar
22-02-18  10:17            27.677 org.eclipse.e4.core.di.extensions.supplier_0.15.0.v20170407-0928.jar
22-02-18  10:17             5.050 org.eclipse.e4.core.di.extensions_0.15.0.v20170228-1728.jar
22-02-18  10:17            46.915 org.eclipse.e4.core.services_2.1.0.v20170407-0928.jar
18-02-18  17:31             1.244 org.eclipse.e4.ui.bindings.nls_0.12.1.v20170823-1632.jar
22-02-18  10:17            43.435 org.eclipse.e4.ui.bindings_0.12.1.v20170823-1632.jar
22-02-18  10:17           194.706 org.eclipse.e4.ui.css.core_0.12.101.v20170712-1547.jar
18-02-18  17:31             1.244 org.eclipse.e4.ui.css.swt.nls_0.13.1.v20170808-1940.jar
22-02-18  10:17           241.074 org.eclipse.e4.ui.css.swt_0.13.1.v20170808-1940.jar
22-02-18  10:17            10.423 org.eclipse.e4.ui.di_1.2.100.v20170414-1137.jar
22-02-18  10:17           363.659 org.eclipse.e4.ui.model.workbench_2.0.1.v20170713-1800.jar
22-02-18  10:17            18.141 org.eclipse.e4.ui.services_1.3.0.v20170307-2032.jar
22-02-18  10:17             8.987 org.eclipse.e4.ui.widgets_1.2.0.v20160630-0736.jar
18-02-18  17:31             3.244 org.eclipse.e4.ui.workbench.addons.swt.nls_1.3.1.v20170319-1442.jar
22-02-18  10:17           143.731 org.eclipse.e4.ui.workbench.addons.swt_1.3.1.v20170319-1442.jar
18-02-18  17:31             4.494 org.eclipse.e4.ui.workbench.renderers.swt.nls_0.14.101.v20170713-1343.jar
22-02-18  10:17           247.611 org.eclipse.e4.ui.workbench.renderers.swt_0.14.101.v20170713-1343.jar
18-02-18  17:31             5.581 org.eclipse.e4.ui.workbench.swt.nls_0.14.101.v20170710-1119.jar
22-02-18  10:17           158.044 org.eclipse.e4.ui.workbench.swt_0.14.101.v20170710-1119.jar
22-02-18  10:17             4.610 org.eclipse.e4.ui.workbench3_0.14.0.v20160630-0740.jar
22-02-18  10:17           239.589 org.eclipse.e4.ui.workbench_1.5.1.v20170815-1446.jar
22-02-18  10:17         1.155.405 org.eclipse.emf.ecore_2.13.0.v20170609-0707.jar
22-02-18  10:18            78.409 org.eclipse.equinox.app_1.3.400.v20150715-1528.jar
22-02-18  10:18           109.464 org.eclipse.equinox.common_3.9.0.v20170207-1454.jar
08-03-18  15:30    <DIR>          org.eclipse.equinox.launcher.win32.win32.x86_64_1.1.551.v20171108-1834
22-02-18  10:18            47.287 org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar
22-02-18  10:18            63.714 org.eclipse.equinox.p2.core_2.4.101.v20170906-1259.jar
18-02-18  17:31             5.666 org.eclipse.equinox.p2.engine.nls_2.5.0.v20170319-2002.jar
22-02-18  10:18           198.782 org.eclipse.equinox.p2.engine_2.5.0.v20170319-2002.jar
18-02-18  17:31             4.074 org.eclipse.equinox.p2.metadata.nls_2.3.200.v20170511-1106.jar
22-02-18  10:18           111.373 org.eclipse.equinox.p2.metadata.repository_1.2.401.v20170906-1259.jar
22-02-18  10:18           318.131 org.eclipse.equinox.p2.metadata_2.3.200.v20170511-1106.jar
18-02-18  17:31             6.107 org.eclipse.equinox.p2.operations.nls_2.4.300.v20170511-1106.jar
22-02-18  10:18            68.045 org.eclipse.equinox.p2.operations_2.4.300.v20170511-1106.jar
22-02-18  10:18           120.686 org.eclipse.equinox.p2.repository_2.3.301.v20170906-1259.jar
18-02-18  17:31             9.181 org.eclipse.equinox.p2.ui.sdk.scheduler.nls_1.3.100.v20170418-0708.jar
22-02-18  10:18           111.290 org.eclipse.equinox.p2.ui.sdk.scheduler_1.3.100.v20170418-0708.jar
22-02-18  10:18            11.719 org.eclipse.equinox.p2.updatechecker_1.1.400.v20170106-2125.jar
22-02-18  10:18           172.227 org.eclipse.equinox.registry_3.7.0.v20170222-1344.jar
22-02-18  10:18           932.827 org.eclipse.gef_3.11.0.201606061308.jar
18-02-18  17:31            18.180 org.eclipse.jface.nls_3.13.2.v20171022-1656.jar
22-02-18  10:18         1.013.514 org.eclipse.jface.text_3.12.1.v20171114-1359.jar
22-02-18  10:18         1.033.366 org.eclipse.jface_3.13.2.v20171022-1656.jar
22-02-18  10:18           233.710 org.eclipse.osgi.compatibility.state_1.1.0.v20170516-1513.jar
18-02-18  17:31            18.942 org.eclipse.osgi.nls_3.12.50.v20170928-1321.jar
22-02-18  10:18         1.310.619 org.eclipse.osgi_3.12.50.v20170928-1321.jar
18-02-18  17:31            27.933 org.eclipse.search.nls_3.11.100.v20170515-1603.jar
22-02-18  10:18           536.609 org.eclipse.search_3.11.100.v20170515-1603.jar
18-02-18  17:31             6.959 org.eclipse.swt.nls_3.106.2.v20171129-0543.jar
22-02-18  10:18         2.689.273 org.eclipse.swt.win32.win32.x86_64_3.106.2.v20171129-0543.jar
22-02-18  10:18            13.345 org.eclipse.swt_3.106.2.v20171129-0543.jar
18-02-18  17:31            51.914 org.eclipse.team.ui.nls_3.8.1.v20170515-1133.jar
22-02-18  10:18         1.534.145 org.eclipse.team.ui_3.8.1.v20170515-1133.jar
18-02-18  17:31             9.558 org.eclipse.text.nls_3.6.100.v20170203-0814.jar
22-02-18  10:18           247.970 org.eclipse.text_3.6.100.v20170203-0814.jar
18-02-18  17:31            18.065 org.eclipse.ui.cheatsheets.nls_3.5.100.v20170515-0748.jar
22-02-18  10:18           316.680 org.eclipse.ui.cheatsheets_3.5.100.v20170515-0748.jar
18-02-18  17:31             7.917 org.eclipse.ui.console.nls_3.7.1.v20170728-0806.jar
22-02-18  10:18           190.906 org.eclipse.ui.console_3.7.1.v20170728-0806.jar
18-02-18  17:31            47.742 org.eclipse.ui.editors.nls_3.11.0.v20170202-1823.jar
22-02-18  10:18           597.396 org.eclipse.ui.editors_3.11.0.v20170202-1823.jar
22-02-18  10:18            90.080 org.eclipse.ui.ide.application_1.2.0.v20170512-1452.jar
18-02-18  17:31           137.235 org.eclipse.ui.ide.nls_3.13.1.v20170822-1526.jar
22-02-18  10:18         2.436.873 org.eclipse.ui.ide_3.13.1.v20170822-1526.jar
18-02-18  17:31             7.653 org.eclipse.ui.navigator.resources.nls_3.6.1.v20170823-1443.jar
22-02-18  10:18           156.358 org.eclipse.ui.navigator.resources_3.6.1.v20170823-1443.jar
18-02-18  17:31            21.461 org.eclipse.ui.nls_3.109.0.v20170411-1742.jar
18-02-18  17:31             8.621 org.eclipse.ui.views.log.nls_1.2.100.v20170515-1458.jar
22-02-18  10:18           153.605 org.eclipse.ui.views.log_1.2.100.v20170515-1458.jar
18-02-18  17:31             5.679 org.eclipse.ui.views.nls_3.9.0.v20170226-1833.jar
22-02-18  10:18            97.222 org.eclipse.ui.views_3.9.0.v20170226-1833.jar
18-02-18  17:31           106.776 org.eclipse.ui.workbench.nls_3.110.1.v20170704-1208.jar
18-02-18  17:31            48.671 org.eclipse.ui.workbench.texteditor.nls_3.10.100.v20170426-2021.jar
22-02-18  10:18           624.754 org.eclipse.ui.workbench.texteditor_3.10.100.v20170426-2021.jar
22-02-18  10:18         3.534.770 org.eclipse.ui.workbench_3.110.1.v20170704-1208.jar
22-02-18  10:18           431.924 org.eclipse.ui_3.109.0.v20170411-1742.jar
18-02-18  17:31             6.909 org.jkiss.dbeaver.core.application.nls_4.3.5.jar
22-02-18  10:18           162.652 org.jkiss.dbeaver.core.application_4.3.5.jar
18-02-18  17:31            87.803 org.jkiss.dbeaver.core.nls_4.3.5.jar
22-02-18  10:18         4.075.272 org.jkiss.dbeaver.core_4.3.5.jar
18-02-18  17:31             5.232 org.jkiss.dbeaver.ext.erd.nls_1.0.9.jar
22-02-18  10:18           260.147 org.jkiss.dbeaver.ext.erd_2.0.79.jar
18-02-18  17:31            13.936 org.jkiss.dbeaver.ext.postgresql.nls_1.0.9.jar
22-02-18  10:18           443.103 org.jkiss.dbeaver.ext.postgresql_2.1.50.jar
22-02-18  10:18           420.103 org.jkiss.dbeaver.ext.sample.database_1.0.14.jar
18-02-18  17:31             7.775 org.jkiss.dbeaver.model.nls_1.0.9.jar
22-02-18  10:18         1.154.093 org.jkiss.dbeaver.model_1.0.66.jar
22-02-18  10:18             8.290 org.jkiss.dbeaver.ui_4.2.11.jar
22-02-18  10:18            76.479 org.jkiss.utils_2.1.60.jar

**Comment by ruspl-afed** [comment]
It looks like you have required org.eclipse.core.runtime bundle that was reported unresolved
Probably it uses another Java VM instance avaivable on your PC
There was a similar issue some time ago https://github.com/dbeaver/dbeaver/issues/1842
You can try to specify your JavaVM location with dbeaver.ini

> -vm C:\jdk1.8.0_51\bin\javaw.exe 

You can see an example here: https://wiki.eclipse.org/Eclipse.ini

**Comment by Pandorius** [comment]
Well it looks like my problem was actually very stupid:

I unpinned my program from the taskbar and then pinned it back. And it now works just fine when I click the button... 

The button was therefore the issue. Each time I reinstalled the program there was no problem because the program was automatically launched after the installation process. That's only when rebooting my computer and subsequently using the taskbar to relaunch the program that the problem occurred.

Thank you for your help!
